F/FMUComplianceChecker/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FMUComplianceChecker"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/modelica-tools/FMUComplianceChecker/archive/refs/heads/master.zip", "2f8f0754164c7c13f8de76f8ec3271899d4650a3ffe895c2e813cd0b46ae7cb7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add subversion
+mkdir ${bindir}/
+cd FMUComplianceChecker-*/   
+mkdir build; cd build
+cmake ..
+make install test
+if [[ "${target}" == *linux* ]]; then     mv ../install/fmuCheck.linux64 ${bindir}/fmuCheck; fi
+if [[ "${target}" == *mingw* ]]; then     mv ../install/fmuCheck.win64.exe ${bindir}/fmuCheck.exe; fi
+chmod +x ${bindir}/*
+LIC_DIR="${prefix}/share/licenses/${SRC_NAME}"
+mkdir -p "${LIC_DIR}"
+mv "../LICENCE" "${LIC_DIR}/LICENSE"
```

Comment:
This is again in documentation: https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file
```suggestion
install_license "../LICENCE"
```